cabot/settings.py

```diff
@@ -318,6 +318,13 @@
 
 # Image for service page
 SERVICE_IMAGE = os.environ.get('SERVICE_IMAGE', None)
+
+# Metricsapp settings
+# Timeout for Elasticsearch queries
+ELASTICSEARCH_TIMEOUT = os.environ.get('ELASTICSEARCH_TIMEOUT', 20)
+# msearch max_concurrent_searches
+ELASTICSEARCH_MAX_CONCURRENT_SEARCHES = os.environ.get('ELASTICSEARCH_MAX_CONCURRENT_SEARCHES', None)
```

Comment:
this one's a little weird since it can be none (for the default concurrency setting)...I did it with code in settings.py :( but lmk if there's a better way